src/js/containers/covid19/assistanceListing/CFDAOpportunityTotals.jsx

```diff
@@ -0,0 +1,82 @@
+
+const propTypes = {
+    code: PropTypes.string
+};
+
+const CFDAOpportunityTotals = ({ code }) => {
+    const [loading, setLoading] = useState(true);
+    const [totals, setTotals] = useState(null);
+    const [error, setError] = useState({ error: false, message: '' });
+    const apiRequest = useRef(null);
+    useEffect(() => {
+        apiRequest.current = fetchOpportunityTotals(code);
+        const asyncFunc = async () => {
+            try {
+                const { data } = await apiRequest.current.promise;
+                setTotals(data);
+                setLoading(false);
+            }
+            catch (e) {
+                if (!isCancel(e)) {
+                    setError({ error: true, message: e.message });
+                    setLoading(false);
+                }
+                console.log(' Error CFDA Opportunities Totals : ', e.message);
+            }
+        };
+        asyncFunc();
+        return () => {
+            if (apiRequest.current) apiRequest.current.cancel();
+        };
+    }, []);
+
+    return (
+        <div className="cfda-opportunities__container">
+            {!error.error &&
+                <div className="cfda-opportunities__totals">
+                    <div className="cfda-opportunities__total">
+                        <div className="cfda-opportunities__total__title">Open</div>
+                        <div className="cfda-opportunities__total__count">
+                            {loading && <div className="dot-pulse" />}
+                            {!loading && totals.posted}
+                        </div>
+                    </div>
+                    <div className="cfda-opportunities__total">
+                        <div className="cfda-opportunities__total__title">Closed</div>
+                        <div className="cfda-opportunities__total__count">
+                            {loading && <div className="dot-pulse" />}
+                            {!loading && totals.closed}
+                        </div>
+                    </div>
+                    <div className="cfda-opportunities__total">
+                        <div className="cfda-opportunities__total__title">Archived</div>
+                        <div className="cfda-opportunities__total__count">
+                            {loading && <div className="dot-pulse" />}
+                            {!loading && totals.archived}
+                        </div>
+                    </div>
+                    <div className="cfda-opportunities__total">
+                        <div className="cfda-opportunities__total__title">Forecasted</div>
+                        <div className="cfda-opportunities__total__count">
+                            {loading && <div className="dot-pulse" />}
+                            {!loading && totals.forecasted}
+                        </div>
+                    </div>
+                </div>
+            }
+            {error.error && <div>{error.message}</div>}
```

Comment:
Can we watch for a 404 here? It seems like a pretty common response and right now it pretty regularly shows:

```Request failed with status of 404```

Would be nice if we could watch for this and say something like, "No data for this CFDA" or just explain why a 404 is expected (assuming that it is for some CFDAs...)